index.html

```diff
@@ -122,5 +121,46 @@ <h3>JavaScript</h3>
         </div>
       </section>
     </main>
+    <h1 class="about_title">About</h1>
+    <div class="about">
```

Comment:
- [ ] I recommend you stick to using semantic HTML at all times. Change this `div` tag to a `section` tag. This will enclose all the about setion.